docs/search/search-query-language/search-operators/threatlookup.md

```diff
@@ -0,0 +1,128 @@
+---
+id: threatlookup
+title: threatlookup Search Operator
+sidebar_label: threatlookup
+---
+
+The `threatlookup` search operator allows you to search logs for matches in [threat intelligence indicators](/docs/platform-services/threat-intelligence/threat-intelligence-indicators/), providing security analytics to help you to detect threats in your environment. 
```

Comment:
I wanted to double-check that this isn't camel-cased (i.e., threatLookup) like the other search operators.

Reply:
This one is not camel-cased. Not all of them are camel-cased. For example, `threatip` isn't.